utilities/transactions/write_unprepared_txn.cc

```diff
@@ -393,8 +415,9 @@ Status WriteUnpreparedTxn::RollbackInternal() {
   // DB in one shot. min_uncommitted still works since it requires capturing
   // data that is written to DB but not yet committed, while the roolback
   // batch commits with PreReleaseCallback.
-  s = db_impl_->WriteImpl(write_options_, &rollback_batch, nullptr, nullptr,
-                          NO_REF_LOG, !DISABLE_MEMTABLE, &seq_used, ONE_BATCH,
+  s = db_impl_->WriteImpl(write_options_, rollback_batch.GetWriteBatch(),
+                          nullptr, nullptr, NO_REF_LOG, !DISABLE_MEMTABLE,
+                          &seq_used, rollback_batch.SubBatchCnt(),
```

Comment:
Why ONE_BATCH is replaced with SubBatchCnt? Do not we do duplicate detection anymore? Also is ONE_BATCH variable unused now?

Reply:
I removed duplicate detection when I converted to vector (since we're storing duplicated data at at that point anyway). We could do dup detect on writing rollback, but we'd need to waste some extra memory to dedup, and I feel like that'll be less useful in phase 2 anyway.

Reply:
ONE_BATCH is still used further down in the same function.